manual/biomes.rst

```diff
@@ -0,0 +1,79 @@
+
+If we assume for just a moment that the value of **-gv** is 1 then the Temperature function is relatively simple. It is a straight line that runs from **-go** to 1 as Temperature runs from 0 to 1. Thus, on the default settings (**-go** = .2) we would multiply the original Humidity value by .2 when Temperature is 0 and we would multiply it by 1 when the Temperature is 1. If we were to use a straight line (**-gv** = 1) then we would multiply by .6 when Temperature is .5, .4 when the Temperature is .25, etc. We do this because without the offset value (**-go**) we find that we are multiplying by numbers that are too small at the coldest end of the Temperature scale resulting in too much **Polar Desert** terrain.
+
+While this offset gives us better results in the colder ranges we find that unfortunately it has a tendency to push the average rainfall up a bit too much in the middle ranges. In order to correct for this we use the following function:
+
+.. math::
+  f(Temperature) = Temperature ^{GV}
+
+where "GV" is the **-gv** variable. This is the same basic function that is used in `gamma correction <https://en.wikipedia.org/wiki/Gamma_correction>`_ and so we have appropriated the term "gamma value" and "gamma offset" to describe our variables (although to be technical our function is not actually a gamma curve).
+
+It should be noted for technical reasons that the curve is actually calculated first, then compressed and shifted by amounts determined by the offset variable (**-go**). This means that the value for f(Temperature) will always range from the offset variable to 1.
+
+To ensure that the **-gv** and **-go** variables are producing a good curve it may be desirable to generate a scatter plot when the planet is being created.
+
+.. image:: https://raw.githubusercontent.com/Mindwerks/worldengine-data/master/images/examples/scatter_plot_example.png
+   :align: center
+
+Each point on this plot is a single point on the landmass of the planet. The Temperature runs across the bottom, ranging from **Polar** to **Tropical** while the Humidity runs along the side, ranging from **Superarid** to **Superhumid**. Lines are drawn to show the dividing point between various Temperature and Humidity ranges. While the current routine does not label the rows and columns the following image is provided to help understand how they correspond to the chart:
+
+.. image:: https://raw.githubusercontent.com/Mindwerks/worldengine-data/master/images/examples/scatter_plot-labelled.png
+   :align: center
+
+Cells in grey are ones which do not technically occur in the standard Holdridge life zone model and as the example above shows there are certain areas of terrain in the example that are both **Polar** and **Semiarid**. Since there are not too many of them we will not concern ourselves as Worldengine will simply treat them as **Polar** and **Arid** (i.e. classify them as **Polar Ice**).
+
+Biome.png
+---------
```

Comment:
Yes, the image is just a screenshot of biomes.html. Unfortunately RST doesn't appear to support coloring the backgrounds of table cells so I couldn't just create the table in the document.

Biomes.png isn't a good title, since the output might not be a png, but just Biomes is a little vague. I think Biome Images would be a better title.